pkg/reconciler/resource/deployment_test.go

```diff
@@ -103,6 +103,9 @@ func TestNewDeploymentWithDefaultContainer(t *testing.T) {
 					Tolerations: []corev1.Toleration{{
 						Key: "taint", Operator: "Exists",
 					}},
+					NodeSelector: map[string]string{
+						"disktype": "ssd",
+					},
```

Comment:
Not sure ho this work, I cannot find the expected selector at the deployment that is being tested. Is it being set somewhere else? Same for the Service version.

Reply:
Should be fixed now